### PR TITLE
Log the Apps start commands and environment parameters

### PR DESCRIPTION
**Pull request by tzolov**

### CODE REVIEW

**Comment by mminella** [comment]
Why is this only applied for apps and not tasks?

File: spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java [hunk 1]
```diff
@@ -327,17 +327,25 @@ public int getInstanceNumber() {
 		 * to the 'out' and 'err' streams of this process.
 		 */
 		private void start(ProcessBuilder builder, Path workDir) throws IOException {
+			if (logger.isDebugEnabled()) {
+				logger.info("Local Deployer Commands: " + String.join(",", builder.command())
```
Comment (markpollack) [inline spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java:331]: log at debug typo

**Comment by markpollack** [comment]
changed .info to .debug and did a reformat of the file. 
Merged as a4d623b7ac781fb903e3a5c5449becdc2ee38129

**Comment by markpollack** [comment]
By mistake, I did not merge this into master.  @mminella good point wrt to the same info being logged for task.

**Comment by markpollack** [comment]
Now merged as 0031f47683d05ea83dafa2a2504dd2ae3e8d5769

Also added logging for task

**Pull request closed without merging**